talk-list.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE HTML>
+<html lang = "en">
+<head>
+  <title>Super Fun Friends Zone</title>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="styles/normalize.css" />
+  <link rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Open+Sans|Righteous">
+  <script src="https://use.fontawesome.com/67c33ef9bc.js"></script>
+  <link rel="stylesheet" href="styles/main.css" />
+
+</head>
+<body>
+  <h1>Super Fun<br>Friends Zone</h1>
+  <h2>Cool Friends Who Teach Each Other Cool Stuff</h2>
+  <main>
+    <div class="intro">
+      <h3><span class="sffz">SFFZ Talk About Things Conference - Featured Talks</span></h3>
+    </div>
+    <div class="divider"></div>
+
+    <div class="talk-list">
+      <div class="talk">
+        <h3 class="title">
+          What Are Birds?
+        </h3>
+        <h4 class="speaker">
+          Carly Ho
+        </h4>
+        <div class="description">
+          Birds are a hugely varied and fascinating part of our world's ecosystem, from common sparrows to parrots to eagles to cranes to weird muppet-looking birds like the potoo. In this talk, I'll showcase a few of my favorite birds and their interesting qualities, and discuss some ways we can help make the world a better place for our feathered friends.
+        </div>
+      </div>
+      <div class="divider"></div>
+      <div class="talk">
+        <h3 class="title">
+          "This Mountain Is a Complicated Metaphor for Nationalism": The Eurovision Song Contest for Beginners
+        </h3>
+        <h4 class="speaker">
+          Cat Cefalu, Benjamin Dionysus
+        </h4>
+        <div class="description">
+          The Eurovision Song Contest--part Battle of the Bands, part sporting event, and part international intrigue-- is largely unknown to Americans. Full of its own tropes, running jokes, and age-old grudges, it's an amazing annual experience that needs more love. Cat and Benjamin have hosted ESC-watching parties for several years running and are excited to share this fandom with their friends.
+        </div>
+      </div>
+      <div class="divider"></div>
+      <div class="talk">
+        <h3 class="title">
+          Baking: A practical and intuitive approach
+        </h3>
+        <h4 class="speaker">
+          Esa Sclafani
+        </h4>
+        <div class="description">
+          Baking is a skill that we often see as being esoteric and scientific in nature.  The concept of precise proportions and measurements is intimidating to most people, even those who are otherwise comfortable in the kitchen. I, as a professional baker, wish to detail how baking doesn't need to be scary, or as precise as we've been led to believe.  If possible, I would love to talk for five minutes and answer questions for five minutes, in case people have specific queries.
+        </div>
+      </div>
+      <div class="divider"></div>
+      <div class="talk">
+        <h3 class="title">
+          word? word. | Emojis & Communication
```

Comment:
I think you need to escape this ampersand here

Reply:
Most modern browsers handle ampersands just fine without escaping, but I'll escape it just in case.